tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/remote/HttpRequestBuilderExtensions.kt

```diff
@@ -42,6 +42,18 @@ fun HttpRequestBuilder.parameterPage(page: Int) {
     parameter("page", page)
 }
 
+fun HttpRequestBuilder.parameterFirstAirDateYear(year: Int?) {
```

Comment:
The extensions are more specific for the search endpoint. 
I would move the methods into the TmdbSearchApi.

Reply:
Make sense.
Let me update it